test/wallet-http-test.js

```diff
@@ -85,6 +85,30 @@ for (const witnessOpt of witnessOptions) {
       addr = Address.fromString(str, node.network);
     });
 
+    it('should enable seed phrase recovery', async () => {
+      const options = {
+        passphrase: 'PASSPHRASE',
+        mnemonic: 'zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo wrong'
+      };
+
+      const testwallet = await wclient.createWallet(`test_seed-${witnessOpt}`, options);
+      assert.strictEqual(testwallet.master.encrypted, false);
+
+      const master1 = await wclient.getMaster(`test_seed-${witnessOpt}`);
+      assert.strictEqual(master1.encrypted, false);
+      assert.strictEqual(master1.mnemonic.phrase, 'zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo wrong');
```

Comment:
I think it's ok to replace the test string here with the variable `options.mnemonic` (same as below)